josh-mongo/index.js

```diff
@@ -61,36 +60,42 @@ class JoshProvider {
       throw new Error('Keys should be strings or numbers.');
     }
     this.db.update({
-      _id: key
+      _id: key,
     }, {
       _id: key,
-      value: val
+      value: val,
     }, {
-      upsert: true
+      upsert: true,
     });
+    return this;
   }
 
   get(key) {
     console.log(`Retrieving ${key}'s data`);
     return this.db.findOne({
-      _id: key
+      _id: key,
     });
   }
```

Comment:
`#inc` seems to use `#get` with a path too, but seems like this doesn't even have that parameter. I don't know how to go about adding that since I haven't got an idea how `Collection#findOne` works.

Reply:
Yea I don't think path is necessary, fixed in [20028be](https://github.com/eslachance/josh-providers/pull/2/commits/20028be9bbe97167f52e352f7a3d229bd080f251)